### PR TITLE
InfoSec Page and Replacement of undraw images

### PR DESCRIPTION
**Pull request by DylanMiller765**
## What type of PR is this? (check all applicable)

- [X] ♻️ Refactor
- [X] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR adds an InfoSec page and replaces the undraw images with more fitting pictures.

## Mobile & Desktop Screenshots/Recordings

About Us Page
InfoSec Page
Tutoring Page

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by DylanMiller765** [comment]
Should the logo on the About Us page be moved to the right? For the Info Sec page is there a rough idea of what should be on it? I checked the Figma and didn't see a page for it.

File: src/components/Navbar/Navbar.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const NavBar = () => {
 					{/* when hackathon will return to active it again no need to delete */}
 					{/* <ActiveListItem url='/hackathons' name='Hackathons' /> */}
 					<ActiveListItem url='/contactus' name='Contact Us' />
+					<ActiveListItem url='/InfoSec' name='InfoSec' />
```
Comment (Brodypen) [inline src/components/Navbar/Navbar.js:40]: url should be /infosec to be consistent with current style

File: src/pages/InfoSec/InfoSec.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+import webgif from '../../assets/CougarCS_Banner.png';
+import { MetaData } from '../../components/Meta/MetaData';
+import './InfoSec.css';
+
+const meta = {
+	title: 'Info-Sec',
+	desc: 'Learn more about CougarCS Web-Dev',
+	url: 'https://cougarcs.com/web-dev',
```
Comment (Brodypen) [inline src/pages/InfoSec/InfoSec.js:10]: change to be accurate to current site.
desc: 'Learn more about CougarCS Info-sec'
url: 'https://cougarcs.com/infosec'

File: src/App.js [hunk 3]
```diff
@@ -43,6 +44,7 @@ const App = () => {
 						<Route path='/gallery/:event' element={<GalleryEvent />} />
 						<Route path='/404' element={<NotFound />} />
 						<Route path='*' element={<FourOFour />} />
+						<Route path='/InfoSec' element={<InfoSec />} />
```
Comment (Brodypen) [inline src/App.js:47]: move route between tutoring and gallery/event

File: src/components/Navbar/Navbar.js (hunk 1 above)
Comment (Brodypen) [reply src/components/Navbar/Navbar.js:40]: info sec should be before contact us in the navbar.

File: src/pages/InfoSec/InfoSec.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+							className='img-fluid'
+							style={{ borderRadius: '10px' }}
+						></img>
+					</Col>
+					<Col md='6' className='order-md-0'>
+						<div className='about-us'>
+							<h1>InfoSec at CougarCS</h1>
+							<p className='lead-text'>
+								Information Security something maybe put team members below not
+								to sure what the vision is here
```
Comment (Brodypen) [inline src/pages/InfoSec/InfoSec.js:32]: look at webdev/tutoring for help on making this page to look better.

Discord button should link to info sec discord.

**Review by Brodypen — CHANGES_REQUESTED** [review]

**Review by Brodypen — APPROVED** [review]

**Pull request merged**